fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -614,7 +610,6 @@ struct EncryptedRangeFileWriter : public IRangeFileWriter {
 	                                           Arena* arena) {
 		Reference<AsyncVar<ClientDBInfo> const> dbInfo = cx->clientInfo;
 		TextAndHeaderCipherKeys cipherKeys = wait(getEncryptCipherKeys(dbInfo, header, BlobCipherMetrics::RESTORE));
-		ASSERT(cipherKeys.cipherHeaderKey.isValid() && cipherKeys.cipherTextKey.isValid());
```

Comment:
nit: any reason we don't want this assert?

Reply:
I did this change in this PR to not fetch header cipher key if authentication is disabled. after that the assert is not valid anymore.